adafruit_dht.py

```
@@ -160,7 +160,7 @@ def _get_pulses_bitbang(self):
             timestamp = time.monotonic()  # take timestamp
             dhtval = True  # start with dht pin true because its pulled up
             dhtpin.direction = Direction.INPUT
-            dhtpin.pull = Pull.UP
```

dont delete existing code, you can catch and ignore an exception isntead

---

Changes committed, sorry for the misunderstanding on my end.